numba/core/ir.py

```diff
@@ -1462,9 +1462,11 @@ def dump(self, file=None):
                     raise ValueError(msg)
                 else:
                     from pygments import highlight
-                    from pygments.lexers import DelphiLexer as lexer
+                    from numba.misc.dump_style \
+                        import NumbaIRLexer as lexer, by_colorscheme
```

Comment:
Perhaps split this as:
```python
from numba.misc.dump_style import NumbaIRLexer as lexer
from numba.misc.dump_style import by_colorscheme
```
the code base layout was recently refactored and it was found that refactoring tools seemed to struggle with more complex spellings like in the above.